config/locales/decidim-awesome/fr.yml

```diff
@@ -0,0 +1,90 @@
+---
+fr:
+  activemodel:
+    attributes:
+      config:
+        allow_images_in_full_editor: Autoriser les images dans l'éditeur HTML
+        allow_images_in_markdown_editor: Autoriser les images dans l'éditeur Markdown
+        allow_images_in_proposals: Autoriser les images dans l'éditeur de proposition
+        allow_images_in_small_editor: Autoriser les images dans l'éditeur HTML minimal
+        use_markdown_editor: Utiliser l'éditeur Markdown à la place de l'éditeur HTML
+      constraint:
+        component_id: ou spécifiquement dans
+        component_manifest: Seulement pour les fonctionnalités du type
+        participatory_space_manifest: Appliquer sur les espaces participatifs de type
+        participatory_space_slug: Seulement dans
+  decidim:
+    admin:
+      menu:
+        decidim_awesome: Decidim Awesome
+    decidim_awesome:
+      admin:
+        checks:
+          index:
+            decidim-admin: Pour le module Admin
+            decidim-core: Pour le module Core
+            decidim-proposals: Pour le module Propositions
+            decidim_version: Version de Decidim %{version}
+            title: Vérification de la compatibilité système
+        config:
+          constraints:
+            add_condition: Ajouter une condition
+            always: Toujours
+            delete: Supprimer
+            edit: Modifier
+            title: 'Actif seulement dans les cas suivants:'
+          experimental: Options expérimentales
+          form:
+            help:
+              allow_images_in_full_editor: Ajouter une icône d'envoi d'image dans tous les éditeurs WYSIWYG avec une barre d'outil complète.
+              allow_images_in_markdown_editor: Permettre à l'éditeur Markdown d'envoyer des images par glisser/déposer ou par copier/coller.
+              allow_images_in_proposals: Permettre l'envoi d'image dans l'éditeur de proposition, pour tous les utilisateurs.
+              allow_images_in_small_editor: Ajouter une icône d'envoi d'image dans tous les éditeurs WYSIWYG avec une barre d'outil minimale.
+              drag_and_drop_supported: Support du glisser/déposer lors de l'envoi d'image. Les images seront envoyées sur le serveur et traitées comme des ressources externes (sans encodage Base64)
+              use_markdown_editor: Remplace l'éditeur WYSIWYG Quill par un éditeur Markdown coté admin et un moteur de rendu HTML coté utilisateur (le texte sera stocké au format Markdown)
+          rich_text_editor_in_public_views: "NOTE: l'éditeur de texte riche est activé pour les participants. Cette option sera donc sans effet. Utilisez plutôt les \"editors hacks\" pour ajouter des images dans les textes des propositions"
+          show:
+            title: Tweaks for %{setting}
```

Comment:
Hi @MoretS,

Is it voluntary to keep this key in English ? 

```
            title: Tweaks for %{setting}
```

Reply:
+1